### DISCUSSION TITLE
HTTPS on V1 layout

### DISCUSSION THREAD

**Discussion by royalswe** [vikejs/vike#825] [Discussion] [answered]
How do I enable HTTPS on server when developing?
I'm working from this layout https://github.com/brillout/vite-plugin-ssr/tree/main/examples/vue-router-v1 
because I already have a finished spa with vue-router.
I saw this repo: https://github.com/aral/vite-plugin-ssr-with-tls but seems old.

I need https because it´s required from the API I fetch data from.
The client side I already setup https with mkcert package.

Thanks
roYal

**Comment by brillout** [comment]
It's more a question related to your server than related to VPS as described in https://vite-plugin-ssr.com/https.

Contribution very much welcome to write a simpler and more up-to-date example.

**Comment by royalswe** [comment]
Thanks for your quick answer @brillout 
Is there any examples where I can use both vue-router and the server part?
Otherwise I will do my best with existing examples.

Thanks

**Reply by brillout** [reply, marked as answer]
Yea, you'll have to manually assemble `examples/vue-router/` with one of the server examples:
```bash
$ cd examples/
$ grep -R '\<express\>' --exclude-dir node_modules --exclude-dir dist
```

It's fairly easy though.
